pkg/version/version.go

```diff
@@ -23,3 +32,87 @@ func MaybePrintVersionAndExit(printVersion bool) {
 	}
 	log.Infof("running conduit version %s", Version)
 }
+
+type versionStatusChecker struct {
+	version string
+	client  pb.ApiClient
+}
+
+func (v versionStatusChecker) SelfCheck() []*healthcheckPb.CheckResult {
+	cliVersion := v.version
+	cliIsUpToDate := &healthcheckPb.CheckResult{
+		Status:           healthcheckPb.CheckStatus_OK,
+		SubsystemName:    "conduit-version",
+		CheckDescription: "CLI is up-to-date",
```

Comment:
What do you think about extracting those strings as constants? I think we did it for all other checkers, e.g. https://github.com/runconduit/conduit/blob/b788daf429a05288bd63ce107928d599a292a2d4/pkg/k8s/api.go#L19